### PR TITLE
doc: add multi schedulers design doc

### PR DESCRIPTION
**Pull request by zen-xu**
ref: #1397 #1391
Signed-off-by: ZhengYu, Xu <zen-xu@outlook.com>

### CODE REVIEW

**Comment by Thor-wl** [comment]
I'll take part in the design and development of this feature. Let's work together. @zen-xu

**Comment by zen-xu** [comment]
> I'll take part in the design and development of this feature. Let's work together. @zen-xu

👌

File: docs/design/multi-schedulers.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+# Multi schedulers
+
+## Introduction
+
+Sometimes we may want to the cluster be divided into multi sections for different usage.
```
Comment (Thor-wl) [inline docs/design/multi-schedulers.md:5]: As the feedback from the users, another scenario is to schedule workloads under ns `kube-system`  by default-scheduler and others by Volcano. So i suggest to take it into consideration, too.

**Comment by Thor-wl** [comment]
I'll merge this doc and continue to complete it.

**Pull request merged**